tutorials/get-started-python/index.mdx

```diff
@@ -0,0 +1,453 @@
+    - We put an `if` statement inside an `if statement`, to say that if there are `too_many_pets` it should print text to say so, AND then do some further `if/elif/else` tests to choose a string to print with a recommendation about rehoming. The final `else` statement is carried out if the initial `if too_many_pets` condition was found to be untrue.
+
+7. Tell Python to run the code in `animals.py` by entering the following command. Make sure you replace the path to the file with the path you noted in step 3:
+
+    ```
+    python3 /home/user/animals.py
+    ```
+
+    The following output displays:
+
+    ```
+    You have too many pets!
+    You should rehome some dogs
+    ```
+
+## Summary
+
+Time to recap everything we've learnt in this tutorial:
+
+**Python Interactive Shell**: to code directly in the Python Interactive Shell, enter `python3` in your terminal and you're ready to go! Enter your input statements, and Python will produce an output based on your code. Type `exit()` to leave the interactive shell.
+
+**Coding in a script**: to write your code in a script, save it in a text file with the `.py` extension and then enter `python3 path/to/script/myfile.py` in the terminal to run the script.
+
+**Print statements**: "printing" in Python just means displaying an output. The print statement syntax is `print("hello world!")`. If you're printing a number, boolean or variable, you should omit the quotation marks.
+
+**Variable types and assignment**: to assign a value to a variable, use the `=` operator. Depending on the syntax we use, Python knows what type of variable to create. `a = 1` creates an int, `a = 1.5` creates a float, `a = "1"` creates a string, `a = True` creates a boolean, `a = [1, 2, 3]` creates a list. You can change the value of a variable after creating by just repeating the `a =` statement and finishing with the new value you want.
+
+**If statements**: use `if` statements to test whether certain conditions are true, and tell Python to vary the next actions accordingly. You can choose to add `elif` statements to add further conditions to test for and conditional actions to carry out, and/or an `else` statement to show what to do if the `if` condition isn't met. The comparison operators `>`, `<`, `>=`, `<=`, `==` and `!=` are particularlyl useful to use when creating conditions in `if` and `elif` statements.
+
+Don't hesitate to play around with all the code examples in this tutorial and see how changing the code changes the output! For example...
+  - What happens if you use the `+` operator to add two string variables together, and print the result?
+  - Can you modify the "cats and dogs" if/else statement to include other animals and print other outputs like "Your pet is lonely, get more pets!" if there is only one cat or dog?
+  - Can you write an if/else statement that tests whether any number is odd or even and prints a statement like `<number> is an even number`? Hint: try using the modulo operator.
```

Comment:
is there any content we could add for a "going further" do you think it's overkill?

Reply:
I did think about that, but since this is really a "total beginners" tutorial, any of the other Python content we have is going to be way too much of a leap. If/when I write the next Python tutorial/s (about loops and collection types), I will add a link here.

Reply:
yes, true 👍🏻 np!